app/controllers/api/conversion_hosts_controller.rb

```diff
@@ -30,6 +30,9 @@ class ConversionHostsController < BaseController
     def create_resource(type, id, data)
       raise BadRequestError, "resource_id must be specified" unless data['resource_id']
       raise BadRequestError, "resource_type must be specified" unless data['resource_type']
+      raise BadRequestError, "auth_user must be specified" unless data['auth_user']
+      raise BadRequestError, "conversion_host_ssh_private_key must be specified" unless data['conversion_host_ssh_private_key']
+      raise BadRequestError, "vmware_vddk_package_url or vmware_ssh_private_key must be specified" unless data['vmware_vddk_package_url'] || data['vmware_ssh_private_key']
```

Comment:
There can be only one param between `vmware_vddk_package_url` and `vmware_ssh_private_key`. IIUC, it will not raise if both are provided. Could you split it please ?

Reply:
The WebUI does not allow to provide both values at a time.  If that's the case, how can both values be provide?